.env

```diff
@@ -1,7 +1,6 @@
-RUST_NIGHTLY=nightly-2022-12-15
 PRODUCTION_RUSTFLAGS="-Ccodegen-units=1 -Clto=on -Cembed-bitcode=yes"
```

Comment:
:art: This also seems unsed. `rg -. PRODUCTION_RUSTFLAGS` shows it only in the `.env`.